src/Columns/tests/gtest_column_vector.cpp

```diff
@@ -0,0 +1,90 @@
+#include <typeinfo>
+#include <Columns/ColumnsNumber.h>
+#include <Common/randomSeed.h>
+#include <gtest/gtest.h>
+
+
+using namespace DB;
+
+static pcg64 rng(randomSeed());
+static constexpr int error_code = 12345;
+static constexpr size_t TEST_RUNS = 500;
+static constexpr size_t MAX_ROWS = 10000;
+static constexpr size_t filter_ratios[] = {1, 2, 5, 11, 32, 64, 100, 1000};
```

Comment:
Suggest to use an std::array (or std::vector) for filter_ratios ... could just call .size() instead of calculating it manually (l. 14).